editor/gui/editor_quick_open_dialog.h

```diff
@@ -31,6 +31,7 @@
 #ifndef EDITOR_QUICK_OPEN_DIALOG_H
 #define EDITOR_QUICK_OPEN_DIALOG_H
 
+#include "core/string/fuzzy_search.h"
```

Comment:
This can be forward declared.

Reply:
Sorry I may not be versed enough with c++ but do you mean to just include the incomplete types here instead of the header? Can that still work if I'm using the type for one of the members?

Reply:
Yes, if the member is a pointer/Ref, a complete type is not required.